src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -687,7 +687,11 @@ public static String systemGetProperty(String property) {
    */
   public static String systemGetEnv(String env) {
     try {
-      return System.getenv(env);
+      String value = System.getenv(env);
```

Comment:
why do we need to check both places?

Reply:
is extracting value from the map works in all OS ? maybe we can just use this approach ?

Reply:
@sfc-gh-akolodziejczyk I let both ways to avoid punishing the OS where it works with the normal mechanism. So only if the environment variable was not resolved with the normal approach it will try the other.

Reply:
@sfc-gh-dprzybysz Can I resolve this comment?